gateway/api/v1/views.py

```diff
@@ -11,7 +11,6 @@
 from api.models import Program, Job, RuntimeJob
 from api.permissions import IsOwner
 from . import serializers as v1_serializers
-from . import services as v1_services
```

Comment:
this is maybe more for a later refactoring (change wasn't introduced in this PR, so let's not change now), but do we call non-v1 serializers here? if not, given that we're already in a `v1/` directory the prefix is a little redundant

Reply:
Yep, I agree. It could be simplified.